projects/pgai/pgai/vectorizer/configuration.py

```diff
@@ -62,13 +61,25 @@ class ChunkingRecursiveCharacterTextSplitterConfig(SQLArgumentMixin):
     arg_type: ClassVar[str] = "chunking"
     function_name: ClassVar[str] = "ai.chunking_recursive_character_text_splitter"
 
-    chunk_column: str
     chunk_size: int | None = None
     chunk_overlap: int | None = None
     separators: list[str] | None = None
     is_separator_regex: bool | None = None
 
 
+@dataclass
```

Comment:
how was this missing before and things worked?

Reply:
I think litellm via alembic just isn't possible right now., there is no test for it either.
We only merged litellm recently, and I think forgot this step.
Tbf both litellm and alembic were in development at the same time, so it wasn't 100% clear who would merge first and the other one would have to update this. I think I mentioned this to james, but with so many things to keep track of it can be forgotten anyway.